sympy/printing/str.py

```diff
@@ -119,11 +122,11 @@ def _print_Dict(self, expr):
 
     def _print_RandomDomain(self, d):
         try:
-            return 'Domain: ' + self._print(d.as_boolean())
+            return 'Domain: (' + self._print(d.as_boolean()) + ')'
```

Comment:
```
>>> where(X > 0)
Domain: 0 < x₁ ∧ x₁ < ∞
>>> latex(where(X > 0))
Domain: 0 < x_{1} \wedge x_{1} < \infty
```
pretty and latex printers don't have the external parenthesis. Maybe we should be consistent with that.